### PR TITLE
add rules UBTU-20-010045

### PR DESCRIPTION
**Pull request by dexterle**
#### Description:

- Update `sshd_use_approved_kex_ordered_stig` to include ubuntu 20.04

#### Rationale:

- According to DISA Canonical Ubuntu 20.04 LTS Security Guide, Version 1, and Revision 8 contains a new rule which specifies: "The Ubuntu operating system SSH server must be configured to use only FIPS-validated key exchange algorithms."

### CODE REVIEW

File: linux_os/guide/services/ssh/ssh_server/sshd_use_approved_kex_ordered_stig/ansible/shared.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# platform = Red Hat Enterprise Linux 7,Oracle Linux 7,multi_platform_sle
```
Comment (dodys) [inline linux_os/guide/services/ssh/ssh_server/sshd_use_approved_kex_ordered_stig/ansible/shared.yml:1]: please use `multi_platform_ubuntu`

File: linux_os/guide/services/ssh/ssh_server/sshd_use_approved_kex_ordered_stig/bash/shared.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# platform = Red Hat Enterprise Linux 7,Oracle Linux 7,multi_platform_sle
```
Comment (dodys) [inline linux_os/guide/services/ssh/ssh_server/sshd_use_approved_kex_ordered_stig/bash/shared.sh:1]: please use `multi_platform_ubuntu`

**Review by dodys — CHANGES_REQUESTED** [review]
a few issues
but just a heads up that we cannot merge this, as this is from revision 8
the profile is still at revision 1

File: linux_os/guide/services/ssh/ssh_server/sshd_use_approved_kex_ordered_stig/bash/shared.sh (hunk 2 above)
Comment (dexterle) [reply linux_os/guide/services/ssh/ssh_server/sshd_use_approved_kex_ordered_stig/bash/shared.sh:1]: Applied suggestion.

**Comment by dexterle** [comment]
> a few issues but just a heads up that we cannot merge this, as this is from revision 8 the profile is still at revision 1

Thanks for the heads up, I didn't realize that the profile was still at revision 1... I've removed the changes to the `stig.profile`.

**Review by dodys — APPROVED** [review]
LGTM, thanks!

**Pull request merged**